man/systemd.exec.xml

```diff
@@ -1305,6 +1309,14 @@
                 <entry>Debugging, performance monitoring and tracing functionality (<citerefentry project='man-pages'><refentrytitle>ptrace</refentrytitle><manvolnum>2</manvolnum></citerefentry>, <citerefentry project='man-pages'><refentrytitle>perf_event_open</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls)</entry>
               </row>
               <row>
+                <entry>@file-system</entry>
+                <entry>Basic file system access: opening and creating files and directories for read and write, reading file properties</entry>
+              </row>
+              <row>
+                <entry>@file-system-ops</entry>
+                <entry>File system operations: more advanced file system operations, such as renaming or removing files and directories, or creating hard and symbolic links.</entry>
```

Comment:
While reading this, I was thinking about some sandbox architectures where the sandboxed application only have access to syscalls that can operate on a file descriptor, but no access to syscall that take a path. An external privileged agent gives all the file descriptors to the sandboxed application in order for it to work.

Maybe a syscall group with only filesytem syscalls using a file descriptor could be interesting?